perception/autoware_tensorrt_rtmdet/config/rtmdet.param.yaml

```diff
@@ -0,0 +1,26 @@
+/**:
+  ros__parameters:
+    # publish debug image, including bounding box, class name, mask
+    is_publish_debug_image: false
+
+    # publish color mask for result visualization
+    is_publish_color_mask: false
+
+    model_path: "$(var data_path)/tensorrt_rtmdet/$(var model_name).onnx"
+    color_map_path: "$(var data_path)/tensorrt_rtmdet/color_map.csv"
+    plugin_paths: [ "./build/autoware_tensorrt_rtmdet/libautoware_tensorrt_rtmdet_plugin.so" ]
+    mean: [ 103.53, 116.28, 123.675 ]
```

Comment:
Please describe the detail.
Consider changing the param's name.

Reply:
Thank you! I've updated the names and added comments.

related commits: https://github.com/autowarefoundation/autoware.universe/pull/8165/commits/56d1f3139c115793b2b8b77405fdd4104bc08e61, https://github.com/autowarefoundation/autoware.universe/pull/8165/commits/f25368e5f322617432818132f27e0be42b9a2a94, https://github.com/autowarefoundation/autoware.universe/pull/8165/commits/1ae9c24efbd4fa73ddc4274766a85f5b0bbb7c1b